RELEASE_NOTES.md

```diff
@@ -1,5 +1,18 @@
 # Release Notes
 
+## 1.9.2
+### Bugfixes
+- Fixed a bug where `podman save` would fail when the target image was specified by digest ([#5234](https://github.com/containers/libpod/issues/5234))
+- Fixed a bug where rootless containers with ports forwarded to them could panic and dump core due to a concurrency issue ([#6018](https://github.com/containers/libpod/issues/6018))
+- Fixed a bug where rootless Podman could race when opening the rootless user namespace, resulting in commands failing to run
+- Fixed a bug where HTTP proxy environment variables forwarded into the container by the `--http-proxy` flag could not be overridden by `--env` or `--env-file` ([#6017](https://github.com/containers/libpod/issues/6017))
+- Fixed a bug where rootless Podman was setting resource limits on cgroups v2 systems that were not using systemd-managed cgroups (and thus did not support resource limits), resulting in containers failing to start
+
+### Misc
+- Rootless containers will now automatically set their ulimits to the maximum allowed for the user running the container, to match the behavior of containers run as root
+- The default Podman CNI network configuration now sets HairpinMode to allow containers to access other containers via ports published on the host
+- Updated containers/common to v0.8.4
+
```

Comment:
We don't mention that libpod.conf has been removed and replaced anywhere that I can see in the release notes.  It would probably be good to add a line here like:

- The libpod.conf file has been removed and Podman now uses the /etc/containers/containers.conf file for rootfull Podman or `$HOME/.config/containers/containers.conf` for rootless Podman.

Reply:
Wasn't this covered in podman-1.9.0?

Reply:
Not explicitly.  We noted the containers.conf and that it added flexibility, but we did not say it replaced libpod.conf.  I think we should note that somewhere, if that means touching up the older note, I'm fine with that.

Reply:
Added a line about this, PTAL